.github/scripts/release.sh

```diff
@@ -18,7 +18,7 @@ NAME="$VERSION ($DATE)"
 
 CURRENT_TAG=$(git describe --abbrev=0 --tags)
 PREVIOUS_TAG=$(git describe --abbrev=0 --tags $(git describe --abbrev=0)^)
-CHANGELOG=$(git log --no-merges --pretty=%s "$PREVIOUS_TAG".."$CURRENT_TAG" --invert-grep --grep="Bomb" --grep "skip ci" --grep "skip changelog" -i | sed -e 's/^/- /' | sed -E 's/\(#[0-9]+\)//')
+CHANGELOG=$(git log --no-merges --pretty=%s "$CURRENT_TAG".. | egrep -v "skip ci|skip changelog" -i | sed -e 's/^/- /' | sed -E 's/\(#[0-9]+\)//')
```

Comment:
@sderosiaux @gaspb Do we know why there was a difference between CDK script and this script around the `"$PREVIOUS_TAG".."$CURRENT_TAG"`

Here, it was `"$PREVIOUS_TAG".."$CURRENT_TAG"` while in CDK it was only `"$CURRENT_TAG"..`?

Reply:
on master: we work until the latest, here, we work between tags (not the latest which can have more commits already)

Reply:
Ok I change then

Reply:
Changed